tree/tests/cp/mod.rs

```diff
@@ -440,7 +440,8 @@ fn test_cp_preserve_slink_time() {
     let dangle = &format!("{test_dir}/dangle");
     let d2 = &format!("{test_dir}/d2");
 
-    fs::create_dir(test_dir).unwrap();
+    fs::create_dir(test_dir)
```

Comment:
This test was failing to me after a failed test run, seems like a temporary file that didn't get cleaned up after previous aborted run. Perhaps we should make this a little more idempotent.